hdr_html_canvas_element.md

```diff
@@ -435,114 +284,118 @@ function convertExtendedLinearSRGBtoREC2100HLG(r, g, b) {
   const [r5, g5, b5] = hlg_oetf(r4, g4, b4);
-  
+
   return [r5, g5, b5];
 }
 ```
 
-##### Conversion from extended-sRGB to PQ
-
-_Input:_ Full-range non-linear floating-point `extended-srgb` pixel with black at 0.0 and diffuse white at 1.0. Values may exist outside the range 0.0 to 1.0.
-
-_Output:_ Full-range non-linear floating-point `rec2100-pq` pixel with black at 0.0 and diffuse white at ???. Values may exist outside the range 0.0 to 1.0.
+_Note 1:_ As `rec2100-hlg` is a relative format, the brightness of the virtual monitor used for mathematical transforms can be chosen to be any level. In this transform it is chosen to be 302 cd/m2 so that the brightness of diffuse white matches the diffuse white of sRGB (80 cd/m2). Using the extended range gamma formula in footnote 2 of BT.2100, this also sets the HLG Inverse OOTF to be unity. The value 0.265 is calculated by taking the inverse OETF of 0.75, the `rec2100-hlg` diffuse white level.
 
-_Process:_
+_Note 2:_ See section 5.3 in ITU-R BT.2408-4 relating to negative transfer functions in format conversions.
 
-##### Conversion from extended-sRGB to PQ
+##### `srgb-linear` to `rec2100-pq`
 
-_Input:_ Full-range non-linear floating-point `extended-linear-srgb` pixel with black at 0.0 and diffuse white at 1.0. Values may exist outside the range 0.0 to 1.0.
+_Input:_ Full-range non-linear floating-point `srgb-linear` pixel with black at 0.0 and diffuse white at 1.0. Values may exist outside the range 0.0 to 1.0.
 
 _Output:_ Full-range non-linear floating-point `rec2100-pq` pixel with black at 0.0 and diffuse white at ???. Values may exist outside the range 0.0 to 1.0.
 
 _Process:_
 
-##### Conversion from HLG to extended-sRGB
+```javascript
+  function connectionTransferFunction(x) {
+    const c1 =  107 / 128;
+    const c2 = 2413 / 128;
+    const c3 = 2392 / 128;
+    const m1 = 1305 / 8192;
+    const m2 = 2523 / 32;
+    const k = 203;
+    if (x < 0) {
+      return 0;
+    } else if (x <= 1) {
+      const p = Math.pow(x, 1 / m2);
+      return (10000 / k) * Math.pow((p - c1) / (c2 - c3 * p), 1 / m1);
+    } else {
+      return (10000 / k);
+    }
+```
+
+_Note:_ The factor `k` is selected such that a display luminance of 203 cd/m<sup>2</sup> results in a linear color value of 1 in `srgb-linear`.
 
-_Input:_ Full-range non-linear floating-point `rec2100-hlg` pixel with black at 0.0 and diffuse white at 0.75. Values may exist outside the range 0.0 to 1.0.
+##### `srgb-linear` to `rec2100-pq`
+
+_Input:_ Full-range non-linear floating-point `extended-srgb` pixel with black at 0.0 and diffuse white at 1.0. Values may exist outside the range 0.0 to 1.0.
 
-_Output:_ Full-range non-linear floating-point `extended-srgb` pixel with black at 0.0 and diffuse white at 1.0. Values may exist outside the range 0.0 to 1.0.
+_Output:_ Full-range non-linear floating-point `rec2100-pq` pixel with black at 0.0 and diffuse white at ???. Values may exist outside the range 0.0 to 1.0.
```

Comment:
What to say for ??? 
PQ diffuse white is at whatever the creative intent places it at, so maybe instead say peak small-area white (10,000 cd/m2) at 1.0?